bin/compute-adam-jars.sh

```diff
@@ -22,8 +22,8 @@ ADAM_REPO="$(cd `dirname $0`/..; pwd)"
 
 CLASSPATH=$("$ADAM_REPO"/bin/compute-adam-classpath.sh)
 
-# list of jars to ship with spark; trim off the first and last from the CLASSPATH
+# list of jars to ship with spark; trim off the first from the CLASSPATH
 # TODO: brittle? assumes appassembler always puts the $BASE/etc first and the CLI jar last
```

Comment:
Yeah, I was confused too. Essentially, `appassembler` puts `$BASE/etc` at the start of the classpath --> this maps to `/etc` on my system.